packages/web/services/user.js

```diff
@@ -59,7 +59,11 @@ export const requestToBeReviewer = async (id, { categories } = {}) => {
 		return false;
 	}
 
-	const response = await apiPost('reviewers', { user_id: id, categories });
+	const response = await apiPost('reviewers', {
+		user_id: id,
+		categories,
+		disclaimers: Array.from(Array(15).keys()),
```

Comment:
```suggestion
		disclaimers: Array.from(Array(15).keys()),
```

não entendi a utilidade disso 🤔  pq isso é necessário?

Reply:
Para integrar o font precisamos fazer um get `/disclaimers?type=reviewers` e pegar os ids. 
como não tá feito essa parte ainda, coloquei uma array pra evitar quebrar os testes do front já que a API vai exigir os disclaimers.

Reply:
Isso, a data de criação do registro é a data de aceite